roadmap/implementors-guide/src/types/overseer-protocol.md

```diff
@@ -211,6 +211,8 @@ Other subsystems query this data by sending these messages.
 enum RuntimeApiRequest {
 	/// Get the current validator set.
 	Validators(ResponseChannel<Vec<ValidatorId>>),
+	/// Get the assignments of validators to parachains.
+	DutyRoster(ResponseChannel<DutyRoster>),
```

Comment:
The old `DutyRoster` is incompatible with the `Scheduler` module

Reply:
The information that the backing subsystem wants:
  * Validators-to-groups assignments
  * All scheduled paras (`Scheduler::scheduled`)
  * Upcoming para assignments to each core
  * Availability info about each occupied core

I'd suggest putting this all into one struct and making a runtime API that returns all of it.
The last two fields won't be necessary now but enable higher throughput of paras so it would be good to expose that information.